src/plugins/core/header_visibility.ts

```diff
@@ -31,6 +32,15 @@ export class HeaderVisibilityPlugin extends CorePlugin {
           ? CommandResult.Success
           : CommandResult.TooManyHiddenElements;
       }
+      case "REMOVE_COLUMNS_ROWS":
+        if (!this.hiddenHeaders[cmd.sheetId]) {
```

Comment:
It's written that way (copy pasted from above) because if the sheet is invalid, we will try to access `this.hiddenHeaders[invalidSheetId][something]`, which will crash. I changed it to `if (!this.getters.tryGetSheet(cmd.sheetId))`, but now it isn't that clear _why_ this check is needed. 

I don't mind either way TBH.